elasticdl/python/elasticdl/api.py

```diff
@@ -182,6 +179,16 @@ def _submit_job(image_name, client_args, container_args):
         force_use_kube_config_file=client_args.force_use_kube_config_file,
     )
 
+    container_args = wrap_python_args_with_string(container_args)
+
+    master_client_command = "python -m elasticdl.python.master.main"
```

Comment:
This appears in multiple places as well

Reply:
It only appears here. Others are `python -m elasticdl.python.ps.main` and `python -m elasticdl.python.worker.main` which are responsible to start the ps process and the worker process.

Reply:
You are correct. I was looking at the old code in the diff where this appears in multiple places.